src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --hover-link-color: #00acdc;
   margin: 0;
 }
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
```

Comment:
what is this rule for?
works without it
try adding content to the tag `<main>` and see the result